backend/account/src/main/java/com/htec/account/security/WebSecurity.java

```diff
@@ -56,6 +56,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .permitAll()
                 .antMatchers(HttpMethod.POST, SecurityConstants.PASSWORD_RESET_URL)
                 .permitAll()
+                .antMatchers(HttpMethod.PUT, "/users/update/photo")
```

Comment:
why this ?

Reply:
This stayed by mistake when merging branches